dist/index.html

```diff
@@ -57,12 +53,15 @@ <h4 class="hidden">Click Below to Confirm</h4>
       </div>
       <button class="submit-button" type="submit" disabled=true>Estimate Cost</button>
     </form>
-    <section class="trips">
+    <section class="trips hidden">
       <section class="card"></section>
     </section>
-    <footer>
-      <p id="total-cost-this-year"></p>
-      <p class="spent-this-year">Spent This Year</p>
+    <footer class="hidden">
+      <div class="total">
+        <p id="total-cost-this-year"></p>
+        <p class="spent-this-year">Spent This Year</p>
+      </div>
+      <p class="fee">*includes a 10% travel agent fee</p>
```

Comment:
I probably should also tell my user that this is included in their estimated price.